sssom/sssom.external.context.jsonld

```diff
@@ -307,164 +193,86 @@
-        "Pubchem": "https://pubchem.ncbi.nlm.nih.gov/compound/",
+        "PuRO": "http://purl.org/spar/pro/",
         "RBO": "http://purl.obolibrary.org/obo/RBO_",
-        "RE": "https://reactome.org/content/detail/",
-        "REACT": "https://reactome.org/content/detail/",
-        "REFSEQ_PROT": "https://www.ncbi.nlm.nih.gov/protein/",
-        "RESID": "http://purl.obolibrary.org/obo/RESID_",
-        "RETIRED_EHDAA2": "http://purl.obolibrary.org/obo/EHDAA2_",
+        "RESID": "https://proteininformationresource.org/cgi-bin/resid?id=",
         "REX": "http://purl.obolibrary.org/obo/REX_",
-        "RGD": "http://rgd.mcw.edu/rgdweb/report/gene/main.html?id=",
-        "RHEA": "https://www.rhea-db.org/rhea/",
         "RNAO": "http://purl.obolibrary.org/obo/RNAO_",
         "RO": "http://purl.obolibrary.org/obo/RO_",
-        "RO_proposed_relation": "http://purl.obolibrary.org/obo/RO_",
         "RS": "http://purl.obolibrary.org/obo/RS_",
-        "RXCUI": "https://mor.nlm.nih.gov/RxNav/search?searchBy=RXCUI&searchTerm=",
         "RXNO": "http://purl.obolibrary.org/obo/RXNO_",
-        "SABIO-RK": "http://sabiork.h-its.org/reacdetails.jsp?reactid=",
-        "SAO": "http://purl.obolibrary.org/obo/SAO_",
+        "SAO": "http://uri.neuinfo.org/nif/nifstd/sao",
         "SBO": "http://purl.obolibrary.org/obo/SBO_",
         "SCDO": "http://purl.obolibrary.org/obo/SCDO_",
         "SCTID": "http://www.snomedbrowser.com/Codes/Details/",
-        "SCTID_2010_1_31": "http://www.snomedbrowser.com/Codes/Details/",
-        "SECONDARY_CAS_RN": "https://commonchemistry.cas.org/detail?ref=",
+        "SCoRO": "http://purl.org/spar/scoro/",
         "SEP": "http://purl.obolibrary.org/obo/SEP_",
         "SEPIO": "http://purl.obolibrary.org/obo/SEPIO_",
         "SIBO": "http://purl.obolibrary.org/obo/SIBO_",
-        "SIGNOR": "https://signor.uniroma2.it/relation_result.php?id=",
-        "SMID-DB": "https://smid-db.org/smid/",
-        "SMP": "https://smpdb.ca/view/",
-        "SNOMECT": "http://www.snomedbrowser.com/Codes/Details/",
-        "SNOMED": "http://www.snomedbrowser.com/Codes/Details/",
-        "SNOMEDCT": "http://www.snomedbrowser.com/Codes/Details/",
-        "SNOMEDCTCT": "http://www.snomedbrowser.com/Codes/Details/",
-        "SNOMEDCTCT_2018_03_01": "http://www.snomedbrowser.com/Codes/Details/",
-        "SNOMEDCTCT_2019_03_01": "http://www.snomedbrowser.com/Codes/Details/",
-        "SNOMEDCT_2010_1_31": "http://www.snomedbrowser.com/Codes/Details/",
-        "SNOMEDCT_US": "http://www.snomedbrowser.com/Codes/Details/",
-        "SNOMEDCT_US_2018_03_01": "http://www.snomedbrowser.com/Codes/Details/",
-        "SNOMEDCT_US_2018_09_01": "http://www.snomedbrowser.com/Codes/Details/",
-        "SNOMEDCT_US_2019_03_01": "http://www.snomedbrowser.com/Codes/Details/",
-        "SNOMEDCT_US_2019_09_01": "http://www.snomedbrowser.com/Codes/Details/",
-        "SNOMEDCT_US_2020_03_01": "http://www.snomedbrowser.com/Codes/Details/",
```

Comment:
@cthoyt all of these SNOMED prefixes are now gone from a the bimap; I think this makes sense, but I am wondering if I basically need to switch to an extended prefix map now so that hard when I encounter `SNOMEDCTCT`, I can go to `SCTID` using `curies` new standardise feature. Is that correct?